test/unit/org/apache/cassandra/cql3/ViewComplexTest.java

```diff
@@ -265,8 +267,8 @@ private void testCellTombstoneAndShadowableTombstones(boolean flush) throws Thro
             ColumnFamilyStore cfs = ks.getColumnFamilyStore("mv");
             List<String> sstables = cfs.getLiveSSTables()
                                        .stream()
-                                       .sorted(Comparator.comparingInt(s -> s.descriptor.generation))
-                                       .map(s -> s.getFilename())
+                                       .sorted(Comparator.comparing(s -> s.descriptor.generation, SSTableUniqueIdentifierFactory.COMPARATOR))
```

Comment:
Nit: We can use `SSTableReader.generationComparator`:
```suggestion
                                       .sorted(SSTableReader.generationComparator)
```